frontend/src/components/input.tsx

```diff
@@ -0,0 +1,43 @@
+import React, { ChangeEvent, FormEvent, useCallback, useMemo, useState } from 'react';
+import styles from './input.module.css';
+import RightArrow from '../icons/map-arrow-right.svg';
+import classNames from 'classnames';
+
+export interface InputProps {
+  sendMessage: (message: string) => void;
+}
+
+export const Input = ({ sendMessage }: InputProps) => {
+  const [userInut, setUserInput] = useState<string>('');
+
+  const onChange = useCallback((event: ChangeEvent<HTMLInputElement>) => {
+    setUserInput(event.target.value);
+  }, []);
+
+  const onSend = useCallback(
+    (event: FormEvent<HTMLElement>) => {
+      event.preventDefault();
+      sendMessage(userInut);
+      setUserInput('');
+    },
+    [sendMessage, userInut]
+  );
+
+  const sendDisabled = useMemo(() => userInut.length === 0, [userInut]);
```

Comment:
Is this a typo for `userInput` or something else
```suggestion
  const sendDisabled = useMemo(() => userInput.length === 0, [userInut]);
```

Reply:
is a typo, thanks